src/App.jsx

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import WeatherRequest from './WeatherRequest';
+
+const API_KEY = '39023aff7ae25b6655f72f0935e31fc6';
 
 function App() {
```

Comment:
For the parent App component, I think it would be most efficient to have the geocode API be called here instead and have everything moves to a new .jsx file/component. My thinking was in the App component, we would only implement the search bar (as part of the sidebar) and the Geocode API call which converts the city name of the user input to a longitude and latitude. Then we can pass these coordinates to other components to retrieve the weather data. My current code changes does this but I won't push and create a PR just yet b/c of merge conflicts.

Reply:
Can you clarify what you mean by having everything in the file moved to a new component? I think I feel similarly in that it's a good idea to pass the Geocode API call to other components, and I was anticipating that `WeatherRequest` would take in data from the Geocode API.

Reply:
Yeah that was the same idea where the data from the Geocode API call would be passed to other components. My thinking was that the only thing inside the App() component would be the the Geocode API call because its the foundation to everything we're doing. So essentially, the Geocode API along with the search bar would be in the App() component, and when a user searches for a city, we would store the lat/lon in a useState and pass to other components if needed. Lmk if that makes sense?

Reply:
Actually, ignore all that, I think this is a good approach. I've spent some time looking at how it can interact with the Geocode API and it'll work better than I thought. Great job on this!